src/index.ts

```diff
@@ -1,5 +1,5 @@
 export type { ProvablePure } from './lib/provable/types/provable-intf.js';
-export { Ledger, initializeBindings } from './snarky.js';
+export { Ledger, initializeBindings, Pickles } from './snarky.js';
```

Comment:
wait, why are we exporting the `Pickles` namespace? if there's no specific reason, I'd rather avoid exporting an undocumented and easy-to-misuse low-level API

Reply:
Oh, that is a leftover from testing, thanks for catching that!